### PR TITLE
ADL Swagger update

### PR DESCRIPTION
**Pull request by begoldsm**
This is done on its own for ADL to minimize code churn mixed in with new
autogen builds. I will be submitting an update to the build of codegen later.

Note that tests had to be re-run across the board due to a new API-version, which has resulted in a larger than ideal PR.

### CODE REVIEW

**Comment by markcowl** [comment]
@begoldsm Please clean up the commits.  There are some insturctions on cleaning up commits here: https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md

**Comment by begoldsm** [comment]
@markcowl done. Commits are cleaned up to only those that actually result in changes. Note that the first commit here couldn't be squashed down (update to latest nightly). That will be re-committed in a future change to reduce noise.

**Comment by begoldsm** [comment]
@markcowl can you take a look now? I would like to get this approved today and then submit my PowerShell PR. Thanks!

File: src/ResourceManagement/DataLake.Analytics/DataLakeAnalytics.Tests/TestHelpers/CommonTestFixture.cs [hunk 1]
```diff
@@ -44,8 +44,13 @@ public class CommonTestFixture : TestBase
         public string Location = "East US 2";
         public DataLakeAnalyticsManagementHelper DataLakeAnalyticsManagementHelper { get; set; }
         private MockContext context;
-        public CommonTestFixture(MockContext contextToUse, bool createWasbAccount = false)
+        public CommonTestFixture(MockContext contextToUse, bool createWasbAccount = false, bool isDogfood = false)
         {
+            if(isDogfood)
+            {
```
Comment (markcowl) [inline src/ResourceManagement/DataLake.Analytics/DataLakeAnalytics.Tests/TestHelpers/CommonTestFixture.cs]: This is not a best practice - it creates maintenance headaches, especially as the regions supported in dogfood, and even the regions in Azure that support the service you want change over time.  It's a better practice to use the resource manager library to get the needed provider locatiosn for your service

File: src/ResourceManagement/DataLake.Analytics/Microsoft.Azure.Management.DataLake.Analytics/Generated/Models/USqlCredential.cs [hunk 2]
```diff
@@ -32,45 +32,18 @@ public partial class USqlCredential : CatalogItem
...
-            DatabaseName = databaseName;
-            Identity = identity;
             Name = name;
-            UserName = userName;
         }
 
         /// <summary>
-        /// Gets or sets the name of the database the credential is in.
-        /// </summary>
-        [JsonProperty(PropertyName = "databaseName")]
```
Comment (markcowl) [inline src/ResourceManagement/DataLake.Analytics/Microsoft.Azure.Management.DataLake.Analytics/Generated/Models/USqlCredential.cs]: Breaking changes

**Review by markcowl — COMMENTED** [review]
Breaking changes in:
- DataLakeStore/FireWallRule model

The model changes are the ones that will need to be mitigated in PowerShell.  If they aare used in parameters or ourput types of PowerShell cmdlets

You can choose wehter to mitigate these breaking changes in your sdk or in powershell cmdlets.

File: src/ResourceManagement/DataLake.Analytics/Microsoft.Azure.Management.DataLake.Analytics/Generated/Models/USqlCredential.cs (hunk 2 above)
Comment (begoldsm) [inline src/ResourceManagement/DataLake.Analytics/Microsoft.Azure.Management.DataLake.Analytics/Generated/Models/USqlCredential.cs]: @markcowl I will fix these breaking changes in the SDK. For the firewall rule changes we never did anything with firewall rules in powershell previously, those are net new cmdlets.

File: src/ResourceManagement/DataLake.Analytics/DataLakeAnalytics.Tests/TestHelpers/CommonTestFixture.cs (hunk 1 above)
Comment (markcowl) [inline src/ResourceManagement/DataLake.Analytics/DataLakeAnalytics.Tests/TestHelpers/CommonTestFixture.cs]: added #151 and assigned to you

**Pull request merged**